event-land/scss/_variables.scss

```diff
@@ -0,0 +1,17 @@
+/*------------------------------------*\
+  #COLORS
+\*------------------------------------*/
+
+$primary-color: #50cb8b;
+$primary-color-lighter: #effff7;
+$secondary-color: #ffffff;
+$secondary-color-darker: #f1f1f1;
+$tertiary-color: #24222c;
+
+/*------------------------------------*\
+  #FONT
+\*------------------------------------*/
+
+// font family
+$font-family: "Inter", sans-serif;
```

Comment:
I think, the comment is redundant, you can delete it, the variable is already explain it. 
I recommend to use sth like this: `primaryFontFamily`